components/cli/pkg/util/utils.go

```diff
@@ -584,22 +584,19 @@ func ExecuteCommand(cmd *exec.Cmd, errorMessage string) error {
 	}()
 	err := cmd.Start()
 	if err != nil {
-		fmt.Printf("cellery : %v: %v \n", errorMessage, err)
-		os.Exit(1)
+		ExitWithErrorMessage("Error executing command", err)
 	}
 	err = cmd.Wait()
 	if err != nil {
-		fmt.Printf("cellery : %v: %v \n", errorMessage, err)
-		os.Exit(1)
+		ExitWithErrorMessage("Error executing command", err)
 	}
 	return nil
 }
 
 func DownloadFromS3Bucket(bucket, item, path string, displayProgressBar bool) {
 	file, err := os.Create(filepath.Join(path, item))
 	if err != nil {
-		fmt.Printf("Error in downloading from file: %v \n", err)
-		os.Exit(1)
+		ExitWithErrorMessage("Failed to download from s3 bucket", fmt.Errorf("Error downloading from file", err))
```

Comment:
Shall we append to file name to the error message?

Reply:
Updated.